dist/amd-resolver.js

```diff
@@ -2,6 +2,78 @@
 (function() {
   "use strict";
 
+  var File = require('./file');
```

Comment:
Not really sure why this is saying that anything changed here... I didn't change anything in `src/`